llvm/include/llvm/CodeGen/SelectionDAGISel.h

```diff
@@ -192,7 +192,15 @@ class SelectionDAGISel : public MachineFunctionPass {
     OPC_EmitMergeInputChains1_1,
     OPC_EmitMergeInputChains1_2,
     OPC_EmitCopyToReg,
+    OPC_EmitCopyToReg0,
+    OPC_EmitCopyToReg1,
     OPC_EmitCopyToReg2,
+    OPC_EmitCopyToReg3,
+    OPC_EmitCopyToReg4,
+    OPC_EmitCopyToReg5,
+    OPC_EmitCopyToReg6,
+    OPC_EmitCopyToReg7,
+    OPC_EmitCopyToRegHalf,
```

Comment:
Half -> TwoByte

I assume Half meant half word, but a word isn't standardized as 32 bits. X86 for example use 16-bit for word and 32 bits is double word. There is no such thing has a half word on X86. Therefore using Half here would't be meaningful to some developers familiar with other targets.